src/test/java/org/dependencytrack/notification/publisher/AbstractWebhookPublisherTest.java

```diff
@@ -18,30 +18,26 @@
  */
 package org.dependencytrack.notification.publisher;
 
-import com.github.tomakehurst.wiremock.junit.WireMockRule;
-import org.junit.Before;
-import org.junit.Rule;
-
+import com.github.tomakehurst.wiremock.junit5.WireMockRuntimeInfo;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
 import jakarta.json.JsonObjectBuilder;
+import org.junit.jupiter.api.BeforeEach;
 
 import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
 import static com.github.tomakehurst.wiremock.client.WireMock.anyUrl;
 import static com.github.tomakehurst.wiremock.client.WireMock.post;
 import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
-import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
 import static org.dependencytrack.model.ConfigPropertyConstants.GENERAL_BASE_URL;
 
+@WireMockTest
```

Comment:
Are we creating two WireMock instances by repeating this annotation on `AbstractPublisherTest` and `AbstractWebhookPublisherTest`?